src/Administration/Resources/app/administration/src/module/sw-settings-document/page/sw-settings-document-detail/index.js

```diff
@@ -630,6 +630,10 @@ export default {
         },
 
         onAddDocumentType(type) {
+            if (!this.documentConfig.config.fileTypes) {
+                this.documentConfig.config.fileTypes = [];
```

Comment:
This fix is fine for me and I would leave the additional check here (I don't know where that property gets initialized).

But why we don't initialize the property to an empty array here? Or is it important that it can be `undefined` in some cases?
https://github.com/shopware/shopware/blob/ac0882f849b288cacdf897f921e73c4e72b36ea8/src/Administration/Resources/app/administration/src/module/sw-settings-document/page/sw-settings-document-detail/index.js#L49-L53

Reply:
Good Idea. I also add it to the initial config! :+1: